src/main/java/me/magnet/consultant/ConfigUpdater.java

```diff
@@ -164,4 +183,16 @@ private Properties updateConfig(List<KeyValueEntry> entries) {
 		return properties;
 	}
 
+	public void shutDown() {
+		shutdownBegun.set(true);
+		if (request != null) {
```

Comment:
Since you probably don't want to retry aborts, this is should probably also contain the CAS operation: `shutdownBegun.compareAndSet(false, true)`